inference_lib/src/aqlm/inference_kernels/cuda_kernel.cpp

```diff
@@ -114,16 +123,20 @@ torch::Tensor code2x8_matmat(
     );
   }
   flat_output *= scales.flatten().unsqueeze(0);
+  if (bias.has_value()) {
+    flat_output += bias->unsqueeze(0);
+  }
+
   auto output_sizes = input_sizes.vec();
   output_sizes.pop_back();
   output_sizes.push_back(-1);
-  auto output = flat_output.view(output_sizes);
+  auto output = flat_output.reshape(output_sizes).clone();
```

Comment:
Can you please briefly describe why does this need clone on top of reshape?

Reply:
```python
RuntimeError: Output 0 of _QuantizedMatmulBackward is a view and is being modified inplace. This view was created inside a custom Function (or because an input was returned as-is) and the autograd logic to handle view+inplace would override the custom backward associated with the custom Function, leading to incorrect gradients. This behavior is forbidden. You can fix this by cloning the output of the custom Function.
```

Reply:
In Machine we trust!